app/views/state_file/questions/unemployment/_form.html.erb

```diff
@@ -100,7 +100,6 @@
           <div class="form-group-tight">
             <%= f.cfa_input_field(:unemployment_compensation, t('state_file.questions.unemployment.edit.unemployment_compensation_html'), classes: ["form-width--long"]) %>
             <%= f.cfa_input_field(:federal_income_tax_withheld, t('state_file.questions.unemployment.edit.federal_income_tax_withheld_html'), classes: ["form-width--long"]) %>
-            <%= f.cfa_input_field(:unfillable, t('state_file.questions.unemployment.edit.box_10a_html'), classes: ["form-width--long"], options: { value: state_full_name, disabled: true }) %>
```

Comment:
this field is no longer in the figma designs, so i figured the right move was to delete it rather than keep it with the disabled styles